### ISSUE TITLE
MSI Modern 14 Not Booting

### ISSUE THREAD

**Issue by atharvaunde59** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#11]
Hello Dev,
Thanks for the config, With this, I managed to install the Big Sur on MSI Moden 14 (Config same as Modern 15, Sigle NVME).
Via Recovery, I formatted the drive and installed the Big Sur,
Post that system rebooted and open core (Latest) gave me the following options 
1. Boot (External)
2. macOS Installer
3. openCore.efi
4. Reset NVRAM
On selecting the macOS Installer option post-reboot, it takes me back to the macOS Recovery menu only and never gave an option for 1st setup.

This is my 1st attempt with the Latest Build of Big Sur, Kindly let me know if am missing something!

**Comment by AndresGarciaSobrado91** [comment]
Did you solve the problem?

**Comment by atharvaunde** [comment]
I solved it and using it as a daily driver for a couple of weeks along with heliport,
The only weird issue is getting while botting is that it takes too much time to boot and verbose mode keeps showing repeated spam related to intel wifi and Bluetooth drivers, hence sometimes the boot is fast and sometimes it takes 5 min.

No idea what's going wrong.......

**Comment by AndresGarciaSobrado91** [comment]
I see. Are you using release versions of opencore and all of your kexts?

**Comment by atharvaunde** [comment]
Yes, am using the release version of Opencore. I haven't touched kexts as am using them as-is from the repo because of the same hardware config...

**Comment by atharvaunde** [comment]
![PXL_20211006_051236647](https://user-images.githubusercontent.com/19168610/136144570-c84d6692-1b39-4e88-8c1c-59e48aaf4a66.jpg)
My console is getting this log loop continuously,
the `IntelFirmware` section to be precise. 
Am using latest Stable version of Heliport!

**Comment by AndresGarciaSobrado91** [comment]
Ok, notice that several files in my repo were updated, try updating your EFI. Also, please provide your hardware information because your laptop is not the same as mine

**Comment by atharvaunde** [comment]
Thanks, @AndresGarciaSobrado91 I will give it a try!

I just checked the bottom sticker of my laptop and it says the Model is `Modern 14 A10M`

**Comment by atharvaunde** [comment]
Oh yeh, Mine is by default shipped with an AC9560 card. 
Any idea, what I need to change?

**Comment by AndresGarciaSobrado91** [comment]
Seems to be the same as mine (labeled as AC9560 but real model name is AC 9462 right? )

**Comment by AndresGarciaSobrado91** [comment]
Nice, thank you for your feedback @atharvaunde

**Comment by honestyan** [comment]
> I solved it and using it as a daily driver for a couple of weeks along with heliport, The only weird issue is getting while botting is that it takes too much time to boot and verbose mode keeps showing repeated spam related to intel wifi and Bluetooth drivers, hence sometimes the boot is fast and sometimes it takes 5 min.
> 
> No idea what's going wrong.......
> 
> PS: This is my Private ACC and @atharvaunde59 is my Companie's acc

As same as my issue actually. My issue is on the booting process, it will reboot multiply also clear NVRAM much times to get in to OS. Until now, it wasn't solved yet. Maybe your issue was solved you can share here mate.

**Comment by Dorian1034** [comment]
I also have a modern 14, but my problem is different and I wanted to know if someone here was able to change the intel wifi card to a native broadcom one and get it to work, because in my case it was not successful.

**Comment by atharvaunde** [comment]
> > I solved it and using it as a daily driver for a couple of weeks along with heliport, The only weird issue is getting while botting is that it takes too much time to boot and verbose mode keeps showing repeated spam related to intel wifi and Bluetooth drivers, hence sometimes the boot is fast and sometimes it takes 5 min.
> > 
> > No idea what's going wrong.......
> > 
> > PS: This is my Private ACC and @atharvaunde59 is my Companie's acc
> 
> As same as my issue actually. My issue is on the booting process, it will reboot multiply also clear NVRAM much times to get in to OS. Until now, it wasn't solved yet. Maybe your issue was solved you can share here mate.

Extremely sorry for late reply. 
I missed this thread. 

Now, speaking of you issue, 
I had same problem, which got fixed by using latest version of OC and proper config.plist

**Comment by atharvaunde** [comment]
> I also have a modern 14, but my problem is different and I wanted to know if someone here was able to change the intel wifi card to a native broadcom one and get it to work, because in my case it was not successful.

Umm, not sure as my laptop is natively shipped with Intel Wifi + Bluetooth Card Combo (At least in India afaik, MSI ships all laptops with Intel NICs natively)
